src/rgw/rgw_main.cc

```diff
@@ -653,6 +653,23 @@ static int process_request(RGWRados *store, RGWREST *rest, RGWRequest *req, RGWC
 
   dout(1) << "====== req done req=" << hex << req << dec << " http_status=" << http_ret << " ======" << dendl;
 
+  switch(http_ret/100) {
```

Comment:
move this block to line 649 so that all of the code using http_ret is in one place